README.md

```diff
@@ -120,25 +120,26 @@ AsyncContext value, it needs non-trivial refactor:
 ```js
 const asyncVar = new AsyncContext.Context();
 
-function *gen() {
+function* gen() {
   const span = createSpan();
   yield asyncVar.run(span, () => computeResult());
   yield asyncVar.run(span, () => computeResult2());
   // ...or
-  yield* asyncVar.run(span, function *() {
+  yield* asyncVar.run(span, function* () {
     yield computeResult();
     yield computeResult2();
   });
 }
 ```
 
-`.run(val, fn)` creates a new function body. The new function environment
-is not equivalent to the outer environment and can not trivially share code
-fragments between them. Additionally, `break`/`continue`/`return` can not be
-refactored naively.
+`.run(val, fn)` creates a new function body. The new function environment is not
+equivalent to the outer environment and can not trivially share code fragments
+between them. Additionally, `break`/`continue`/`return` can not be refactored
```

Comment:
```suggestion
between them. Additionally, `super`/`break`/`continue`/`return` can not be refactored
```

Reply:
Actually `super` can be called from an arrow function in a constructor. Just not a regular function expression / declaration.